Description: Look for the hll compiler in /usr/lib instead of /usr/libexec
Author: Laurent Bigonville <bigon@debian.org>
Forwarded: not-needed

--- a/man/man5/semanage.conf.5
+++ b/man/man5/semanage.conf.5
@@ -37,7 +37,7 @@ Specify an alternative store_root path t
 
 .TP
 .B compiler-directory
-Specify an alternative directory that contains HLL to CIL compilers. The default value is "/usr/libexec/selinux/hll".
+Specify an alternative directory that contains HLL to CIL compilers. The default value is "/usr/lib/selinux/hll".
 
 .TP
 .B ignore-module-cache
--- a/src/conf-parse.y
+++ b/src/conf-parse.y
@@ -320,7 +320,7 @@ static int semanage_conf_init(semanage_c
 	conf->store_path = strdup(basename(selinux_policy_root()));
 	conf->ignoredirs = NULL;
 	conf->store_root_path = strdup("/var/lib/selinux");
-	conf->compiler_directory_path = strdup("/usr/libexec/selinux/hll");
+	conf->compiler_directory_path = strdup("/usr/lib/selinux/hll");
 	conf->policyvers = sepol_policy_kern_vers_max();
 	conf->target_platform = SEPOL_TARGET_SELINUX;
 	conf->expand_check = 1;
